manifests/init.pp

```diff
@@ -452,6 +455,20 @@
     }
   }
 
+  if $sshd_config_chrootdirectory != undef {
+    validate_absolute_path($sshd_config_chrootdirectory)
+  }
+
+  if $sshd_config_forcecommand != undef {
+    if is_string($sshd_config_forcecommand) == false {
```

Comment:
validate_string() accepts numbers as string, is_string() does not. Therefore the extra mile here.